R/tostring.R

```diff
@@ -805,14 +1087,159 @@ wrap_string <- function(str, width, collapse = NULL) {
 }
 
 .go_stri_wrap <- function(str, w) {
+  if (is.na(w) || is.nan(w)) {
+    browser()
+  }
+  if (w < 1) {
```

Comment:
Can this happen?

Reply:
hmm. well the top case clearly doesn't happen in any of our tests cause the test suite doesn't hang, lol.

the browser can definitely be taken out. whether w can be 0... I feel like in the truetype wrapping algorithm it can, but I'd have to dig in to be sure, lets explore that wed

Reply:
I think it makes sense to keep it as a safenet, even if it is not a probable scenario